straxen/plugins/veto_hitlets.py

```diff
@@ -159,3 +166,67 @@ def drop_data_field(old_hitlets, new_hitlets):
         n['left'] = o['left']
         n['low_left'] = o['low_left']
         n['record_i'] = o['record_i']
+
+
+@export
+@strax.takes_config(
+    strax.Option(
+        'save_outside_hits_mv',
+        default=(2, 5), track=True,
+        child_option=True, parent_option_name='save_outside_hits_nv',
+        help='Save (left, right) samples besides hits; cut the rest'),
+    strax.Option(
+        'hit_min_amplitude_mv',
+        default=20, track=True,
+        child_option=True, parent_option_name='hit_min_amplitude_nv',
+        help='Minimum hit amplitude in ADC counts above baseline. '
+             'Specify as a tuple of length n_nveto_pmts, or a number.'),
+    strax.Option(
+        'min_split_mv', 
+        default=100, track=True,
+        child_option=True, parent_option_name='min_split_nv',
+        help='Minimum height difference pe/sample between local minimum and maximum, '
+             'that a pulse get split.'),
+    strax.Option(
+        'min_split_ratio_mv',
+        default=0, track=True,
+        child_option=True, parent_option_name='min_split_ratio_nv',
+        help='Min ratio between local maximum and minimum to split pulse (zero to switch this off).'),
+    strax.Option(
+        'entropy_template_mv',
+        default='flat', track=True,
+        child_option=True, parent_option_name='entropy_template_nv',
+        help='Template data is compared with in conditional entropy. Can be either "flat" or an template array.'),
+    strax.Option(
+        'entropy_square_data_mv',
+        default=False, track=True,
+        child_option=True, parent_option_name='entropy_square_data_nv',
+        help='Parameter which decides if data is first squared before normalized and compared to the template.'),
+    strax.Option('gain_model_mv',
+                 child_option=True, parent_option_name='gain_model_nv',
+             help='PMT gain model. Specify as (model_type, model_config)'),
+)
+class muVETOHitlets(nVETOHitlets):
```

Comment:
man, you really nailed it with the `child_plugins`, look at how short this class is!

Reply:
Thanks :)